src/containers/couresCarousel/index.js

```diff
@@ -1,20 +1,22 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import AliceCarousel from 'react-alice-carousel';
+import { IoIosArrowBack, IoIosArrowForward } from 'react-icons/io';
 
 import 'react-alice-carousel/lib/alice-carousel.css';
 import styles from './styles.module.scss';
 
 class CoursesCarousel extends Component {
-  state = { 
-    currentIndex: 0 
-  };
+  state = {};
 
   galleryItems = () => {
     return Array(12)
       .fill()
       .map((item, i) => (
         <div className={styles.course_card_container}>
```

Comment:
So, maybe it is better to destructure styles element in this component because u use many elements like a styles.adding_button, styles.image, styles.name and etc.